### DISCUSSION TITLE
Using resnet network for training seems to have no effect

### DISCUSSION THREAD

**Discussion by wyclearnpy** [talmolab/sleap#1794] [Help!]
I used the resnet network for single-instance training, but the detection results were not good or even useless. Is there something wrong with my configuration or something else? Has anyone encountered the same situation? Below are some of my settings and terminal print information.
![sleap_config1](https://github.com/talmolab/sleap/assets/135505026/4b8c7d03-b3a0-4233-9bf3-c024b7e9db5f)
![sleap_config2](https://github.com/talmolab/sleap/assets/135505026/1b5570da-4a3d-4fb8-b9e2-9dea97e1aca9)
![result](https://github.com/talmolab/sleap/assets/135505026/9ca2bccd-088b-49f2-b540-15fe7dad42ee)
[Terminal print information.txt](https://github.com/user-attachments/files/15544502/Terminal.print.information.txt)

**Comment by roomrys** [comment]
Hi @wyclearnpy,

We always recommend using UNet for the backbone, but if you are set on using ResNet, then try setting the Model > Backbone > Weights to fine tune instead of frozen.

Let us know if that works better.

Thanks,
Liezl
